server/src/main/java/org/apache/iotdb/db/metadata/mtree/MTreeBelowSGMemoryImpl.java

```diff
@@ -916,4 +862,40 @@ protected boolean processFullMatchedMNode(IMNode node, int idx, int level)
   }
 
   // endregion
+
+  // region Interfaces for schema reader
+  public ISchemaReader<INodeSchemaInfo> getNodeReader(IShowNodesPlan showNodesPlan)
```

Comment:
Since the MTreeBelowSGCachedImpl and MTreeBelowSGMemoryImpl are used in different SchemaRegionImpl and there's no scenarios that they serve the same invoker, I think we can remove the interface IMTreeBelowSG in future work.